chapters/snippet-information.md

```diff
@@ -299,16 +294,12 @@ This field contains the license information actually found in the snippet, if an
 
 The options to populate this field are limited to:
 
-The SPDX License List short form identifier, if the license is on the SPDX License List;
-A reference to the license, denoted by LicenseRef-`[idstring]`, if the license is not on the SPDX License List;
+* A valid SPDX License Expression as defined in Annex [D](SPDX-license-expressions.md).
```

Comment:
```suggestion
* A valid SPDX License Expression as defined in [Annex D](SPDX-license-expressions.md).
```

Reply:
@seabass-labrax - I think this is a "nice to have" rather than must fix before merging.   We're in the stage right now of trying to get this release out.   This sort of thing, can be fixed by PR later, rather than blocking a merge now.